test/eventlistener_test.go

```diff
@@ -155,6 +153,19 @@ func TestEventListenerCreate(t *testing.T) {
 		t.Fatalf("Error creating TriggerBinding: %s", err)
 	}
 
+	// ClusterTriggerBinding
+	ctb, err := c.TriggersClient.TektonV1alpha1().ClusterTriggerBindings().Create(
```

Comment:
Can we split this off into a separate test instead of adding onto existing test here since it is already pretty large?

Reply:
So, this is an e2e test, which was using a trigger binding. I have split that into one trigger binding and 1 cluster trigger binding and updated the test. Does it make sense to have two tests for that?

Reply:
I think its ok for now...but I'm planning on adding smaller/more focussed e2e tests soon!